README.md

```diff
@@ -1,3 +1,21 @@
 # Software Orchestration Blueprint
 
-Welcome to the software orchestration blueprint.
+The **Software Orchestration Blueprint** showcases how an example in-vehicle software architecture with connectivity to the cloud can help developers manage complex services and applications that tackle advanced scenarios like workload orchestration, scheduling, placement and resilience. The abstraction layers provided by the blueprint aim to simplify and enhance the developer experience. These include discovery and consumption of resources and services, support for a common vehicle model to be used across vehicles, dynamic topic management for publish and subscribe of vehicle signals and synchronization of signals to a cloud digital twin.
+
+## Sample Use Case
+The blueprint will come with a sample use case. The system detects that a smart trailer is being connected to the car. A signal is raised to the orchestrator that a smart trailer provider and application is required to manage the smart trailer and adjust body functions and powertrain accordingly and ensure a smooth trip. There are safety and security implications with the orchestration of this use case and that will be addressed in a future iteration.
+
+## Architecture
+
+**Implementation with Eclipse Ankaios**
+![Ankaios Implementation](docs/diagrams/ankaios.png)
+
+**Implementation with Eclipse BlueChi**
+![BlueChi Implementation](docs/diagrams/bluechi.png)
```

Comment:
Since the repository is an integration repository for using different software orchestrators and each software orchestrators has its own characteristics. I would keep two separate diagrams.

Reply:
That sounds good to me. I will resolve this conversation now and leave it as we have it.